lib/scrape.js

```diff
@@ -281,25 +321,28 @@ async function fetchOrgsWithData() {
 
   const fetchingAll = orgs.map(async (org, index) => {
     const existingOrg = existingData.find(existing => existing.id === org.id)
-    const fetchingUsers = orgLeaders[index].map(user => {
+    const fetchingUsers = orgLeaders[index].map(async user => {
       if (existingOrg && existingOrg.leaders) {
         const existingUser = existingOrg.leaders.find(
           leader => leader.id === user.id
         )
-        if (existingUser && existingUser.github_account) {
-          return existingUser.github_account
+
+        return await fetchUserFromCache(user, existingUser, orgGitHub[index])
+      } else {
+        return {
```

Comment:
this branch could also use `fetchUserFromCache`, by with `existingUser` being undefined.